modules/nw-disable-http2.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-disable-http2_{context}"]
+= Disable HTTP/2
```

Comment:
Suggest gerund: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings

```suggestion
= Disabling HTTP/2
```